services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -409,6 +409,10 @@ async def _run_scheduler_task(self) -> None:
         settings: DynamicServicesSchedulerSettings = (
             self.app.state.settings.DYNAMIC_SERVICES.DYNAMIC_SCHEDULER
         )
+        logger.debug(
+             "dynamic-sidecars observation interval %s",
+             settings.DIRECTOR_V2_DYNAMIC_SCHEDULER_INTERVAL_SECONDS,
```

Comment:
question: is this not already printed out at the start of director-v2??

Reply:
No this one was not printed.